tests/e2e/test_rosacli_cluster_post.go

```diff
@@ -230,6 +230,27 @@ var _ = Describe("Healthy check",
 				Expect(clusterConfig.Encryption.KmsKeyArn).To(Equal(kmsKey))
 			})
 
+		It("additional allowed principals work on cluster creation - [id:74408]",
+			labels.Critical, labels.Runtime.Day1Post,
+			func() {
+				By("Confirm current cluster profile uses additional allowed principals")
+				if clusterConfig.AdditionalPrincipals == "" {
```

Comment:
Let's not check the clusterConfig, but check the profile enabled or not. Otherwise if the value is not recorded at all we will never find a problem.